relay-config/src/config.rs

```diff
@@ -895,9 +928,15 @@ impl Config {
 
         let config = Config {
             values: ConfigValues::load(&path)?,
-            credentials: match fs::metadata(Credentials::path(&path)) {
-                Ok(_) => Some(Credentials::load(&path)?),
-                Err(_) => None,
+            credentials: if Credentials::path(&path).exists() {
```

Comment:
Please revert to using `fs::metadata` instead of `exists`, since it also checks for access permissions.

Reply:
It just delegates to `fs::metadata()` so it does not do less (or more than it).
I think it makes the code clearer but if you want I can change it back.

```
imp Path {
    ...
    pub fn exists(&self) -> bool {
        fs::metadata(self).is_ok()
    }
```